src/main/app/src/app/admin/formulier-definitie-edit/formulier-definitie-edit.component.html

```diff
@@ -21,32 +21,32 @@
     <form [formGroup]="definitieFormGroup">
       <mat-card>
         <mat-card-header>
-          <mat-card-title>{{'formulierDefinitie'| translate}}</mat-card-title>
+          <mat-card-title>{{ 'formulierDefinitie'| translate }}</mat-card-title>
```

Comment:
Looks like differences in Spotless / Prettier configuration. Should this not already be handled by our Spotless config @hsiliev ? Our Spotless checks do not seem to mind in this case (which is maybe a shame I guess).

Reply:
We do HTML formatting and whitespace is significant in it:
https://prettier.io/blog/2018/11/07/1.15.0#whitespace-sensitive-formatting

Perhaps we need another formatter here